src/data/resolvers/stakes.ts

```diff
@@ -0,0 +1,81 @@
+import { Resolvers } from '@apollo/client';
+import { ClientType, ColonyClientV6, Extension } from '@colony/colony-js';
+import { bigNumberify } from 'ethers/utils';
+
+import { Context } from '~context/index';
+
+export const stakesResolvers = ({
+  colonyManager,
+}: Required<Context>): Resolvers => ({
+  Query: {
+    async stakeMotionLimits(
+      _,
+      { colonyAddress, userAddress, motionId, tokenDecimals },
+    ) {
+      try {
+        const colonyClient = (await colonyManager.getClient(
+          ClientType.ColonyClient,
+          colonyAddress,
+        )) as ColonyClientV6;
+        const votingReputationClient = await colonyClient.getExtensionClient(
+          Extension.VotingReputation,
+        );
+        const {
+          skillRep,
+          stakes,
+          domainId,
+        } = await votingReputationClient.getMotion(motionId);
+        const { skillId } = await colonyClient.getDomain(domainId);
+        const { reputationAmount } = await colonyClient.getReputation(
+          skillId,
+          userAddress,
+        );
+        // @NOTE There's no prettier compatible solution to this :(
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        // eslint-disable-next-line max-len
+        const userMinStakeFraction = await votingReputationClient.getUserMinStakeFraction();
+
+        const totalStaked = bigNumberify(stakes[1])
+          .div(bigNumberify(10).pow(tokenDecimals))
+          .toNumber();
+        const totalStakeAmount = skillRep
+          .mul(totalStakeFraction)
+          .div(bigNumberify(10).pow(tokenDecimals * 2))
+          .toNumber();
+        const userMinStakeAmount = skillRep
+          .mul(totalStakeFraction)
+          .mul(userMinStakeFraction)
+          .div(bigNumberify(10).pow(tokenDecimals * 3))
```

Comment:
For the native token, this 10 to the power of 54!!!!!!!, why ?

This is super wrong

Reply:
Is that so? I asked Alex how could I get the user-readable/ ETH version of the amounts and he explained it to me like this:

> both totalStakeFraction and userMinStakeFraction are stored as fixed-point decimals on chain and wherever you're using either you have to divide by 10 ** 18 to get the 'real' number
> if you're using both, as in the case of minStake, you have to divide by 10 ** 18 twice, or by 10 ** 36

This is my interpretation of that explanation.  It's 10 ^ 54 due to `totakStakeFraction`, `userMinStakeFraction`, and `skillRep`, from what I understand, the 3 of them are stored on 10 ^ 18

Reply:
Same as above, 90% right. Talked to Alex, it should be:
```
        const userMinStakeAmount = skillRep
          .mul(totalStakeFraction)
          .mul(userMinStakeFraction)
          .div(bigNumberify(10).pow(tokenDecimals + 2 * 18))
```